### ISSUE TITLE
flatpak copyq doesnt start minimized

### ISSUE THREAD

**Issue by gituser823** [hluk/CopyQ#2706]
**Describe the bug**
I want copyq to start minimized in system tray.

**To Reproduce**
I found an older bug about this that said to:
1. delete ~/.config/autostart/copyq.desktop
2. run "copyq config autostart true"

I deleted the desktop file and then ran
flatpak run --command="bash" com.github.hluk.copyq
Inside I ran
/app/bin/copyq config autostart true

It said "true", but now it doesnt start at all anymore after a reboot.

**Expected behavior**
Copyq started in maximized mode before, I just want it to start minimized

**Screenshots**
-

**Version, OS and Environment**

CopyQ Clipboard Manager 8.0.0
Qt: 5.15.10
KNotifications: 5.115.0
Compiler: GCC
Arch: x86_64-little_endian-lp64
OS: KDE Flatpak runtime

- OS
LMDE 6
- Desktop environment, window manager
Cinnamon, X11

**Comment by gituser823** [comment]
Well, I have the autostart option already configured in my Desktop environment, that's how it was set up before. 

But I can not tell it to start minimized, because there is no command line option I can set in the start command line parameters executed by the DE.

If you could add a command line option to start minimized, that works with the Flatpak version my issue would be solved.

**Comment by hluk** [comment]
Hmm, the app uses different desktop file when started explicitly from menu or launcher (it basically triggers `copyq --start-server show`) and when auto-started which is runs `copyq` without any arguments (which does not show the main window).

Not sure if I can define different desktop files (or commands) for the flatpak or handle it somehow in the app itself. :thinking:

**Comment by gituser823** [comment]
Did you have time to look into, if you can implement this?

**Comment by hluk** [comment]
> Did you have time to look into, if you can implement this?

Nope. I would welcome any help. I don't know if there is a way to distinguish whether the app was started by the user or using the autostart feature. It would be then possible to modify the desktop file to handle both situations.

**Comment by DebakantPradhan** [comment]
just remove the show from 

`> copyq --start-server show
`
only 
`copyq --start-server`
work fine start this minimised
